docs/docs-ref-autogen/outlook_1_2/outlook/office.appointmentcompose.yml

```diff
@@ -9,7 +9,7 @@ summary: >-
 
   **Important**: This is an internal Outlook object, not directly exposed through existing interfaces. You should treat
   this as a mode of `Office.context.mailbox.item`<!-- -->. For more information, refer to the [Object
-  Model](https://docs.microsoft.com/office/dev/add-ins/reference/objectmodel/requirement-set-1.2/office.context.mailbox.item)
```

Comment:
@AlexJerabek Does the versionremover script need to be updated to handle this link change? The requirement set should match so e.g. this should remain 1.2

Reply:
Yes, good catch.